application/core/QuestionTypes/ArrayMultiscale/RenderArrayMultiscale.php

```diff
@@ -317,15 +318,17 @@ public function parseSubquestionsNoDropdown(&$aData)
             $aData['answertextright'] = $answertextright;
             if ($aData['shownoanswer']) {
                 if (count($aData['labelans1']) > 0) {
-                    if (empty($this->getFromSurveySession($myfname1))) {
+                    $fname1value = $this->getFromSurveySession($myfname1);
+                    if ($fname1value !== '0' && empty($fname1value)) {
```

Comment:
The previous code was easier to understand …

Reply:
Why? This means: not-set  = anaything falsy but different to "0" string

Reply:
That was exactly the case which triggered the error.
Actually, I think this fix is better as changes less.

Reply:
Ehm, the previous line is easier to read,  yes, but it's not correct, is it? :sweat_smile:

Reply:
> The previous code was easier to understand …

Denis means this code: `if (!isset($fname0value) || $fname0value == '') {`
https://github.com/LimeSurvey/LimeSurvey/pull/3061/commits/bd25e53309895520fa8a9b9c0bf94e1009ba0633#r1183785808

Reply:
Yes,

Comment replace the code.

Reply:
Ah ok. Well, no biggie, really, I don't think we need to waste time here. :)

Reply:
I already resolve :)